tests/test_backend.py

```diff
@@ -55,15 +55,9 @@ def setUp(self):
         self.addCleanup(cm.disable)
 
         self.cache = caches["default"]
-        try:
-            self.cache.clear()
-        except Exception:
-            pass
+        self.cache.clear()
```

Comment:
please put it on `tearDown` it is easier to understand and complies with the common practice

Reply:
Done.